cmd/target-browser/cmd/backupplan.go

```diff
@@ -28,18 +27,16 @@ func backupPlanCommand() *cobra.Command {
 }
 
 func runBackupPlan(*cobra.Command, []string) error {
```

Comment:
func name `runBackupPlan` seems misleading.. we're not running backupPlans here 

update all func names wherever necessary

Reply:
Fixed